pdns/packethandler.cc

```diff
@@ -108,23 +108,29 @@ bool PacketHandler::addCDNSKEY(DNSPacket& p, std::unique_ptr<DNSPacket>& r, cons
 {
   string publishCDNSKEY;
   d_dk.getPublishCDNSKEY(p.qdomain,publishCDNSKEY);
-  if (publishCDNSKEY != "1")
+  if (publishCDNSKEY.empty())
     return false;
 
   DNSZoneRecord rr;
-  bool haveOne=false;
+  rr.dr.d_type=QType::CDNSKEY;
+  rr.dr.d_ttl=sd.minimum;
+  rr.dr.d_name=p.qdomain;
+  rr.auth=true;
 
+  if (publishCDNSKEY == "0") { // delete DS via CDNSKEY
+    rr.dr.d_content=std::make_shared<DNSKEYRecordContent>("0 3 0 AA==");
```

Comment:
`"0 3 0 AA=="` and `"0 0 0 00"` are candidates to become two `const static string`

Reply:
with https://www.rfc-editor.org/errata/eid5049 in a comment near them